app/models/pipeline_run.rb

```diff
@@ -347,6 +347,8 @@ def results_finalized?
   end
 
   def ready_for_cache?
+    # TEMPORARILY DISABLE CACHING COMPLETELY - REMOVE EARLY RETURN TO REINSTATE
+    return false
```

Comment:
I'd prefer to see this as an AppConfig so we can still debug in other envs

Reply:
I'd be happy to take over this PR and do that.

Reply:
See https://github.com/chanzuckerberg/idseq-web/pull/3223

Reply:
Ok thanks, please feel free to do that